vgm_trml.c

```diff
@@ -3601,7 +3601,7 @@ static void ShowPlayingNotes(const char* ChipStr, UINT16 ChnCount, UINT32 ChnMas
 	{
 		if ((ChnMask >> CurChn) & 0x01)
 		{
-			ChnPtr += sprintf(ChnPtr, "%u, ", CurChn);
+			ChnPtr += snprintf(ChnPtr, BUFFER_SIZE, "%u, ", CurChn);
```

Comment:
Considering the pointer manipulation here, I think this can still overflow.

Maybe we should just exit the loop once we're in danger of overflow? i.e. `(ChnPtr + 4) > (Buffer + BUFFER_SIZE)`
If so, then `sprintf` vs. `snprintf` wouldn't matter.

Reply:
good catch. will think of a better way to handle this